scene/debugger/scene_debugger.cpp

```diff
@@ -96,7 +96,7 @@ void SceneDebugger::deinitialize() {
 #ifdef DEBUG_ENABLED
 void SceneDebugger::_handle_input(const Ref<InputEvent> &p_event, const Ref<Shortcut> &p_shortcut) {
 	Ref<InputEventKey> k = p_event;
-	if (k.is_valid() && k->is_pressed() && !k->is_echo() && p_shortcut->matches_event(k)) {
+	if (k.is_valid() && k->is_pressed() && !k->is_echo() && p_shortcut.is_valid() && p_shortcut->matches_event(k)) {
```

Comment:
```suggestion
	if (k.is_valid() && p_shortcut.is_valid() && k->is_pressed() && !k->is_echo() && p_shortcut->matches_event(k)) {
```
Minor but might be best to get the null checks out of the way immediately